src/LooksRare.ts

```diff
@@ -296,20 +296,69 @@ export class LooksRare {
    * @param taker Taker order
    * @param signature Signature of the maker order
    * @param merkleTree If the maker has been signed with a merkle tree
-   * @param referrer Referrer address if applicable
+   * @param affiliate Affiliate address if applicable
    * @returns ContractMethods
    */
   public executeOrder(
     maker: Maker,
     taker: Taker,
     signature: string,
-    merkleTree: MerkleTree = { root: constants.HashZero, proof: [] },
-    referrer: string = constants.AddressZero,
+    merkleTree: MerkleTree = defaultMerkleTree,
+    affiliate: string = constants.AddressZero,
     overrides?: Overrides
   ): ContractMethods {
     const signer = this.getSigner();
     const execute = maker.quoteType === QuoteType.Ask ? executeTakerBid : executeTakerAsk;
-    return execute(signer, this.addresses.EXCHANGE_V2, taker, maker, signature, merkleTree, referrer, overrides);
+    return execute(signer, this.addresses.EXCHANGE_V2, taker, maker, signature, merkleTree, affiliate, overrides);
+  }
+
+  /**
+   * Execute several orders
+   * @param orders List of orders data
+   * @param isAtomic Should the transaction revert or not if a trade fails
+   * @param affiliate Affiliate address
+   * @param overrides Call overrides
+   * @returns ContractMethods
+   */
+  public executeMultipleOrders(
```

Comment:
Just wondering why you didn't call this `executeMultipleTakerBids`?

Reply:
Because support for multiple taker asks is coming, and this function will follow the same pattern than `executeOrder`